include/openssl/sm3.h

```diff
@@ -30,10 +30,8 @@ typedef struct SM3state_st {
    unsigned int num;
 } SM3_CTX;
 
-int sm3_init(SM3_CTX *c);
-int sm3_update(SM3_CTX *c, const void *data, size_t len);
-int sm3_final(unsigned char *md, SM3_CTX *c);
+int SM3_Init(SM3_CTX *c);
+int SM3_Update(SM3_CTX *c, const void *data, size_t len);
+int SM3_Final(unsigned char *md, SM3_CTX *c);
 
-void sm3_block_data_order(SM3_CTX *c, const void *p, size_t num);
-
-#endif
+#endif /* HEADER_SM3_H */
```

Comment:
Again, we were not going to expose new algorithms.  This needs to end up somewhere else.